frontend/packages/dev-console/src/components/pipelineruns/PipelineRunResourceList.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { DevPreviewBadge } from '@console/shared';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { RunFilters } from '../pipelines/PipelineRuns';
+import { PipelineRunModel } from '../../models';
+import PipelineRunsList from './PipelineRunList';
+
+interface PipelineRunResourceListProps {
+  namespace: string;
+}
+
+const PipelineRunResourceList: React.FC<PipelineRunResourceListProps> = ({ namespace }) => (
+  <ListPage
+    showTitle
+    canCreate={false}
+    kind={referenceForModel(PipelineRunModel)}
+    namespace={namespace}
+    ListComponent={PipelineRunsList}
+    rowFilters={RunFilters}
+    badge={<DevPreviewBadge />}
```

Comment:
Can we actually get rid of this page? https://github.com/openshift/console/pull/2917 will cover adding this badge generically without the need for an additional file.